tests/integration/interactive_init_test.go

```diff
@@ -210,6 +210,52 @@ var _ = Describe("odo init interactive command tests", func() {
 				})
 			})
 		})
+
+		When("alizer detection of javascript name", func() {
+
+			BeforeEach(func() {
+				helper.CopyExample(filepath.Join("source", "nodejs"), commonVar.Context)
+				Expect(helper.ListFilesInDir(commonVar.Context)).To(
+					SatisfyAll(
+						HaveLen(3),
+						ContainElements("Dockerfile", "package.json", "server.js")))
+			})
+
+			It("should display node-echo name", func() {
+				language := "javascript"
+				projectType := "nodejs"
+				projectName := "node-echo"
+
+				output, err := helper.RunInteractive([]string{"odo", "init"}, nil, func(ctx helper.InteractiveContext) {
+					helper.ExpectString(ctx, "Based on the files in the current directory odo detected")
+
+					helper.ExpectString(ctx, fmt.Sprintf("Language: %s", language))
+
+					helper.ExpectString(ctx, fmt.Sprintf("Project type: %s", projectType))
+
+					helper.ExpectString(ctx,
+						fmt.Sprintf("The devfile \"%s\" from the registry \"DefaultDevfileRegistry\" will be downloaded.", projectType))
+
+					helper.ExpectString(ctx, "Is this correct")
+					helper.SendLine(ctx, "\n")
+
+					helper.ExpectString(ctx, "Select container for which you want to change configuration")
+					helper.SendLine(ctx, "\n")
+
+					helper.ExpectString(ctx, "Enter component name")
+					helper.SendLine(ctx, "\n")
+
+					helper.ExpectString(ctx, fmt.Sprintf("Your new component '%s' is ready in the current directory", projectName))
+				})
+				Expect(err).To(BeNil())
+
+				lines, err := helper.ExtractLines(output)
+				Expect(err).To(BeNil())
+				Expect(len(lines)).To(BeNumerically(">", 2))
+				Expect(lines[len(lines)-1]).To(Equal(fmt.Sprintf("Your new component '%s' is ready in the current directory", projectName)))
```

Comment:
Do we need to add this check again after the check on L248?

Reply:
I don't think so as the previous check covers the integration test, this is just extra.